firmware/main.c

```diff
@@ -99,13 +105,18 @@ uchar usbFunctionSetup(uchar data[8]) {
 
 		/*
 		 * Auto sck probing is done here vs in FUNC_ENABLEPROG
-		 * So that host applications that use FUNC_TRANSMIT instead to
+		 * So that host applications that use FUNC_TRANSMIT to do
 		 * their own raw ISP port access instead of letting the USBasp device
-		 * do it form them, will also benefit from the auto sck probing.
+		 * do it for them will also benefit from the auto sck probing.
 		 */
 
 		if (prog_sck == USBASP_ISP_SCK_AUTO) {
-			prog_sck = autosck(USBASP_ISP_SCK_1500);
+			/*
```

Comment:
> Let me know if you want this change and I'll go do it.
Yes, I think that's better.
I really think we should make an avrdude change request some day...